halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/DeploymentController.java

```diff
@@ -22,69 +22,76 @@
 import com.netflix.spinnaker.halyard.model.v1.Halconfig;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
 import java.util.List;
 import java.util.stream.Collectors;
 
 @RestController
+@NoArgsConstructor
```

Comment:
Ah whoops, the DeploymentController was meant for a later PR and is not ready. Let me fix this up.